.github/workflows/test_pr_and_main.yml

```diff
@@ -612,7 +612,8 @@ jobs:
       if: ${{ ! matrix.slim }}
       shell: bash
       run: |
-        $PYTHON_EXE -m pip install --cache-dir cache/pip highspy \
+        echo "NOTE: temporarily pinning to highspy pre-release for testing"
+        $PYTHON_EXE -m pip install --cache-dir cache/pip "highspy>=1.7.1.dev1" \
```

Comment:
Should we remove 'temporarily' if this is how we're leaving it?

Reply:
I think we should leave it for now. When they cut a new (non-dev) highspy release we can remove the version requirement.